lib/common/util.js

```diff
@@ -600,30 +623,47 @@ function makeAuthorizedRequestFactory(config) {
    *     request options.
    */
   function makeAuthorizedRequest(reqOpts, callback) {
-    if (config.customEndpoint) {
-      // Using a custom API override. Do not use `google-auth-library` for
-      // authentication. (ex: connecting to a local Datastore server)
-      if (callback.onAuthorized) {
-        callback.onAuthorized(null, reqOpts);
-      } else {
-        util.makeRequest(reqOpts, config, callback);
-      }
+    var streamMode = !callback;
+    var stream;
+    var reqConfig = extend({}, config);
 
-      return;
+    callback = callback || util.noop;
```

Comment:
No, that is a great catch! Somewhere along the way, I tried to make the variable naming easier to understand, and I think I failed. I updated the PR. PTAL!